builder/Dockerfile

```diff
@@ -21,17 +21,30 @@ RUN apk --no-cache add alpine-sdk \
                        netcat-openbsd \
                        nodejs \
                        pkgconfig \
-                       py-mysqldb \
-                       py-psycopg2 \
-                       py-yaml \
-                       python \
+                       py-pip \
+                       python3 \
+                       python3-dev \
                        util-linux-dev \
                        zlib-dev \
                        libuv-dev \
                        lz4-dev \
                        openssl-dev \
                        libgcrypt-dev \
-                       cmake
+                       mariadb-dev \
+                       cmake \
+                       mongo-c-driver-dev
+
+# Add Python dependencies from PyPi using `pip` we have not APK(s) for:
+RUN pip install mysqlclient PyYAML
```

Comment:
what for do we need `PyYAML`?

Reply:
I don't know - maybe we can remove this dependency if it is no longer required?

Please note, I added it instead of the ```py-yaml``` alpine package as per request from @prologic (see above).

Reply:
well, i am not sure it was ever required 😄 

we vendor py-yamlm, it was always like so

https://github.com/netdata/netdata/tree/master/collectors/python.d.plugin/python_modules

Reply:
Perhaps @ilyam8 is right :) Legacy, I have no idea either.

Reply:
Thanks for the clarification @ilyam8